botorch/acquisition/joint_entropy_search.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+r"""
+Acquisition function for joint entropy search (JES). The code utilizes the
+implementation designed for the multi-objective batch setting.
+
+"""
+
+from __future__ import annotations
+
+from typing import Any, Optional
+
+from botorch.acquisition.multi_objective.joint_entropy_search import (
+    qLowerBoundMultiObjectiveJointEntropySearch,
+)
+from botorch.acquisition.multi_objective.utils import compute_sample_box_decomposition
+from botorch.models.model import Model
+from botorch.utils.transforms import concatenate_pending_points, t_batch_mode_transform
+from torch import Tensor
+
+
+class qLowerBoundJointEntropySearch(qLowerBoundMultiObjectiveJointEntropySearch):
```

Comment:
It's seems kind of odd to have the single objective version inherit from the multi-objective version (just an observation, not a criticism)

Reply:
I agree, I wasn't too keen on this but it was a bit involved doing it the other way round, especially for PES